src/main/java/frc/robot/RobotConstants.java

```diff
@@ -167,10 +168,16 @@ public static boolean isCompBot() {
 
     public static class PanelManipulator {
 
+        public static final int SPINNER_MOTOR_ID = 5;
+
+        
         /**
-         * Threshold for color comparison
+         * Game Colors
          */
-        public static final double DEFAULT_COLOR_THRESHOLD = 20;
+        public static Color8Bit redGameColor    = new Color8Bit(255, 0, 0);
```

Comment:
Please use proper naming standards for constants (LIKE_THIS)

Also, this should be in fieldconstants, in its own sub-class